crates/hyperqueue/src/dashboard/ui/screens/job/job_info_display.rs

```diff
@@ -0,0 +1,87 @@
+use crate::dashboard::data::job_timeline::DashboardJobInfo;
+use crate::dashboard::ui::styles::table_style_deselected;
+use crate::dashboard::ui::terminal::DashboardFrame;
+use crate::dashboard::ui::widgets::table::{StatefulTable, TableColumnHeaders};
+use crate::transfer::messages::JobDescription;
+use chrono::{DateTime, Local};
+use tui::layout::{Constraint, Rect};
+use tui::widgets::{Cell, Row};
+
+#[derive(Default)]
+pub struct JobInfoTable {
+    table: StatefulTable<JobInfoDataRow>,
+}
+
+#[derive(Default, Debug)]
+struct JobInfoDataRow {
+    pub label: &'static str,
+    pub data: String,
```

Comment:
Since these rows get created frequently, I would suggest using `Cow<str>` instead of `String` for columns that may contain dynamic data.